doc/circuitikzmanual.tex

```diff
@@ -2142,6 +2142,7 @@ \subsubsection{Special sources}
     \circuitdescbip*{vsourcetri}{Triangle voltage source}{tV}
     \circuitdescbip*{esource}{Empty voltage source}{}
     \circuitdescbip*{pvsource}{Photovoltaic-voltage source}{}
+    \circuitdescbip*{pvmodule}{Photovoltaic module source}{}
```

Comment:
You can do here:

    \circuitdescbip*{pvmodule}{Photovoltaic module source\footnotemark}{}
    \footnotetext{Added by André Alves in \texttt{v1.3.5}}

...or whatever you thinks fit

Reply:
Sure, that sounds good. Thanks :-)
Would you like to wrap everything up into a single commit and issue another PR? Or can you compress everything into a single commit? I can get those changes made pretty soon

Reply:
I can squash it. You could also, by using `git rebase -i master` on your side and then force-push,  but if you are not confident with it, I'll do it here on github --- it's just a matter of selecting "squash and merge" instead of just "merge" ;-)

Reply:
If you could add the missing bit soon, better, I'd like to release a 1.3.5 this weekend (a lot of changes!)

Reply:
Missing bits were just pushed to the branch. I was going to take the time to figure out how to squash the commits, but if you already know how to, it will probably be faster